addons/captives/XEH_postInit.sqf

```diff
@@ -35,6 +35,22 @@ if (isServer) then {
 
 if (!hasInterface) exitWith {};
 
+    //Add Keybind:
+["ACE3 Common", QGVAR(captives), [(localize LSTRING(KeyComb)), (localize LSTRING(KeyComb_description))],
+{
+    if !(cursorObject isKindOf "Man") exitWith {};
+
+    if([ACE_player, cursorObject] call FUNC(canApplyHandcuffs)) exitWith {
+        [cursorObject, true] call FUNC(setHandcuffed);
+    };
+
+    if([ACE_player, cursorObject] call FUNC(canRemoveHandcuffs)) exitWith {
+        [cursorObject, false] call FUNC(setHandcuffed);
```

Comment:
Don't use `cursorObject`. It has really bad accuracy.

Reply:
What should I use instead of `cursorObject`?

Reply:
`screenToWorld [0.5, 0.5]` maybe?

Reply:
@commy2 ?